pymmh3.py

```diff
@@ -73,14 +72,345 @@ def fmix( h ):
     if tail_size >= 1:
         k1 ^= key[ tail_index + 0 ]
     
-    if tail_size != 0:
+    if tail_size > 0:
         k1  = ( k1 * c1 ) & 0xFFFFFFFF
-        k1  = ( k1 << 15 | k1 >> 17 ) & 0xFFFFFFFF # _ROTL32
+        k1  = ( k1 << 15 | k1 >> 17 ) & 0xFFFFFFFF # inlined ROTL32
         k1  = ( k1 * c2 ) & 0xFFFFFFFF
         h1 ^= k1
 
+    #finalization
     return fmix( h1 ^ length )
 
+def hash128( key, seed = 0x0, x64arch = True ):
```

Comment:
I saw that now as well, I agree that conformance is more important that anything else. My bad.